RheaAdh/task2/index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>NavBar using FlexBox</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <div class="navbar">
+      
+        <ul>
+          <li><a class="logo">WOOHOO</a></li>
+          <li><a href="#">Home</a></li>
```

Comment:
```suggestion
          <li><a class="logo">WOOHOO</a></li>
          <div style="flex-grow: 1"></div>
          <li><a href="#">Home</a></li>
```
Navbar looks good when it is of full-width.
![2020-05-10 01:34:37](https://user-images.githubusercontent.com/32123917/81483806-868a1980-925e-11ea-846f-9d3ce0538dad.png)

This will create a flexible space between __WOOHOO__ and other nav links. You can change the color of flexible blank space to make it even more attractive.